library/kapacitor

```diff
@@ -20,12 +10,12 @@ GitRepo: git://github.com/influxdata/influxdata-docker
 GitCommit: dec27fb32f5d9696cd0627f3ac97d353f7557b31
 Directory: kapacitor/1.0/alpine
 
-Tags: 1.1.0-rc2
+Tags: 1.1, 1.1.0, latest
```

Comment:
Looks like you've added a new `latest` tag without removing the previous one. :smile: